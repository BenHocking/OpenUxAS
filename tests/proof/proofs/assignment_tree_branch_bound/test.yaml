filenames: ["assignment_tree_branch_bound.ads"]
level: 4
