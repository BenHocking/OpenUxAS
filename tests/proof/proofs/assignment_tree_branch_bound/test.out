assignment_tree_branch_bound.adb:267:08: medium: "All_Actions_In_Map" is recursive, terminating annotation could be incorrect
assignment_tree_branch_bound.adb:454:22: medium: "Prove_TaskOptionId_In_Map" is recursive, terminating annotation could be incorrect
assignment_tree_branch_bound.adb:1652:07: warning: statement has no effect
assignment_tree_branch_bound.adb:1652:18: warning: "Algebra" is set by "Free_Tree" but not used after the call
